src/components/ControlBar/EditBar.js

```diff
@@ -50,9 +56,16 @@ const EditBar = props => {
     }
 
     const onSave = () => {
-        props.onSave().then(newId => {
-            setRedirectUrl(`/${newId}`)
-        })
+        try {
+            props
+                .onSave()
+                .then(newId => {
+                    setRedirectUrl(`/${newId}`)
+                })
+                .catch(() => failureAlert.show())
+        } catch (e) {
```

Comment:
why receive `(e)` if it's not used?
I'm surprised this wasn't caught by your linter...

Reply:
try/catch removed since it wasn't needed (not using async/await)